Documentation/Links.md

```diff
@@ -0,0 +1,72 @@
+# SerenityOS-related pages across the web
+
+This is a roughly categorized list of pages relating to SerenityOS and its subprojects across the web. Many of these pages are "unofficial" in that some single person created them and/or maintains them independently; but we only list pages here that are widely recognized and of good quality. If you want your Serenity related page to be listed, just create a pull request adding it and we can discuss.
+
+## General
+
+-   **[serenityos.org](https://serenityos.org)**
+    -   [Manpages](https://man.serenityos.org/)
+    -   Birthday Posts
+        -   [1st](https://www.serenityos.org/happy/1st/)
+        -   [2nd](https://www.serenityos.org/happy/2nd/)
+        -   [3rd](https://www.serenityos.org/happy/3rd/)
+        -   [4th](https://www.serenityos.org/happy/4th/)
+    -   [Emoji](https://emoji.serenityos.org/)
+-   [Ports](https://ports.serenityos.net/)
+-   [Try it out Online!](https://copy.sh/v86/?profile=serenity)
```

Comment:
Maybe mention that this is an old i686 build, and not necessarily representative of the current state of some parts of the system